src/middlewared/middlewared/plugins/kubernetes_linux/backup.py

```diff
@@ -58,7 +58,8 @@ def backup_chart_releases(self, job, backup_name):
 
             secrets = self.middleware.call_sync(
                 'k8s.secret.query', [
-                    ['type', '=', 'helm.sh/release.v1'], ['metadata.namespace', '=', chart_release['namespace']]
+                    ['type', 'in', ['helm.sh/release.v1', 'Opaque']],
```

Comment:
Good catch there, this explains why the secrets where not being backed up correctly :)